src/NWCClient.test.ts

```diff
@@ -56,3 +56,85 @@ describe("NWCClient", () => {
     expect(nwcClient.options.lud16).toBe("hello@getalby.com");
   });
 });
+
+describe("selectHighestCompatibleVersion", () => {
+  let nwcClient: NWCClient;
+  let selectVersion: (walletVersions: string[]) => string | null;
+  const ORIGINAL_SUPPORTED_VERSIONS = NWCClient.SUPPORTED_VERSIONS;
+
+  beforeEach(() => {
+    nwcClient = new NWCClient();
+    // Access the private method using type assertion
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    selectVersion = (nwcClient as any).selectHighestCompatibleVersion;
+  });
+
+  afterEach(() => {
+    // Restore the original SUPPORTED_VERSIONS
```

Comment:
This feels a bit hacky to me. Maybe can be solved by the above comment. 

Also, the JS SDK knows what versions it supports, I think editing this value and testing it does not really make sense

Reply:
We agreed to not have tests given `selectHighestCompatibleVersion` is now hard-coded/simplified